src/app/space/add-space-overlay/add-space-overlay.component.html

```diff
@@ -42,8 +42,7 @@ <h1>Create a New Space</h1>
                 </div>
                 <div class="form-group">
                   <label for="description" class="control-label">Description</label>
-                  <div contenteditable="true" id="description" name="description" type="text" class="form-control create-launcher-description" placeholder="Enter a Space Description">
-                  </div>
+                  <input id="description" name="description" type="text" class="form-control create-launcher-description" placeholder="Enter a Space Description" #description/>
```

Comment:
The `id` should be unique for the system. While it may work, it is safer to use the page name as a namespace prefix.